packages/grid/data-grid/src/tests/layout.DataGrid.test.tsx

```diff
@@ -64,6 +68,35 @@ describe('<DataGrid /> - Layout & Warnings', () => {
       expect(rect.width).to.equal(400 - 2);
     });
 
+    it('should throw an error if rows props is being mutated. Error should be thrown due to Object.freeze(rows) on rows prop', () => {
```

Comment:
Test cases inside the "Layout" block only run in a real browser with `yarn test:karma`. Since this test doesn't require one, you can move it to outside to be able to run it with `yarn test:unit`.

Reply:
Thanks @m4theushw . I moved it outside.
But I checked that `yarn test:karma` runs tests even if it is outside the `Layout`, `warning` blocks. But `yarn test:unit` runs **only** those tests which are outside those blocks. 
So by moving outside, it is still running within the browser. 

Also curious regarding where is that kind of configuration setup?